File: codeforces/Archive/A/996A/solution.cpp
#include <iostream>

using namespace std;

int main()
{
    int n = 0;
    int count = 0;
    cin >> n;
    
    int nums[5] = {100, 20, 10, 5, 1};
    
    for (int i=0; i<5; i++){
        if (n>0){
            count += (n/nums[i]);
            n -= (n/nums[i])*nums[i];
        }
    }
  
    cout << count << '\n';
    return 0;
}

File: codeforces/Archive/A/996A/solution.py
n = int(input())
count = 0
nums = [100, 20, 10, 5, 1]
    
for i in range(5):
    if n > 0:
        count += (n//nums[i])
        n -= (n//nums[i])*nums[i]

print(count)
